### PR TITLE
Python 3.9 build for OpenSSL 3 [skip ci]

### PR DESCRIPTION
**Pull request by ryanskeith**
- Updated build number
- Updated recipe to not use cmake dependent on openssl
- Updated abs.yaml to go to specific channel

### CODE REVIEW

**Comment by JeanChristopheMorinPerso** [comment]
With my last commit, builds are now passing on all platforms.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -80,6 +80,7 @@ source:
       - patches/0034-have-pyunicode-decodeunicodeescape.patch
       # MailCap CVE (CVE-2015-20107) was fixed in 3.9.16
       # - patches/0035-py39-mailcap-CVE-2015-20107.patch
+      - patches/0036-openssl3.patch
```
[the remark this answers is not in the document]
Comment (cbouss) [reply recipe/meta.yaml:83]: perhaps:
```
      {% if openssl.startswith("3") %}
      - patches/0036-openssl3.patch
      {% endif %}
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -80,6 +80,9 @@ source:
       - patches/0034-have-pyunicode-decodeunicodeescape.patch
       # MailCap CVE (CVE-2015-20107) was fixed in 3.9.16
       # - patches/0035-py39-mailcap-CVE-2015-20107.patch
+{% if '3.0' in openssl %}
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:83]: Maybe `{% if openssl.startswith('3.') %}` would be more reliable and future proof?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -322,6 +325,7 @@ outputs:
         -   popd
         - popd
         - test ! -f default.profraw   # [osx]
+        - python -c "import ssl; assert '{{ openssl }}' in ssl.OPENSSL_VERSION"
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:328]: assert ssl.OPENSSL_VERSION.startswith('{{ openssl }}')`
```suggestion
        - python -c "import ssl; assert ssl.OPENSSL_VERSION.startswith('{{ openssl }}')"
```

File: recipe/meta.yaml (hunk 2 above)
Comment (ryanskeith) [reply recipe/meta.yaml:83]: This fails but there is a fix that is more explicit.

Comment (ryanskeith) [reply recipe/meta.yaml:83]: from @cbouss :

{% if (openssl | string).startswith('3.0') %}

**Pull request merged**